src/index.ts

```diff
@@ -63,6 +64,5 @@ function lro(options: {}) {
 (lro as any).ALL_SCOPES = operationsClient.ALL_SCOPES;
 
 export {lro};
-export {grpc};
-exports.createByteLengthFunction = grpc.createByteLengthFunction;
-exports.version = require('../../package.json').version;
+export const createByteLengthFunction = GrpcClient.createByteLengthFunction;
+export const version = require('../../package.json').version;
```

Comment:
Since you're changing it, would you mind switching to [pkginfo](https://www.npmjs.com/package/pkginfo)? Having `../../package.json` in TypeScript code breaks webpack in a weird way (even though this code is never executed in browser context, the `require` still triggers an error).

Reply:
(I'm not webpacking gax right now but it will eventually happen :) )

Reply:
Let's do that in a follow up PR :)